pkg/controllers/resources/persistentvolumeclaims/syncer.go

```diff
@@ -1,8 +1,12 @@
 package persistentvolumeclaims
 
 import (
+	"context"
+	"github.com/loft-sh/vcluster/pkg/controllers/resources/persistentvolumes"
 	synccontext "github.com/loft-sh/vcluster/pkg/controllers/syncer/context"
 	"github.com/loft-sh/vcluster/pkg/controllers/syncer/translator"
+	"github.com/pkg/errors"
```

Comment:
Nit-pick: this package is in maintenance mode, we could drop it completely if we use 
`fmt.Errorf("something, %s", err)` instead of `errors.Wrap`
but it's super minor :)